extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -160,6 +161,10 @@ export function getElement(selector, el, selectionMethod) {
     return closestByTag(el, selector);
   } else if (selectionMethod == 'scope') {
     return el.parentElement.querySelector(selector);
+  } else if (selectionMethod == 'host') {
+    const elWin = el.ownerDocument.defaultView;
+    const parentEl = elWin.frameElement && elWin.frameElement.parentElement;
+    return closestBySelector(parentEl, '.-amp-element');
```

Comment:
This has to be a `!Element`, right? But it could be null from the `elWin.frameElement` test.